src/Types/Auth.ts

```diff
@@ -73,6 +73,7 @@ export type SignalDataTypeMap = {
 	'app-state-sync-key': proto.Message.IAppStateSyncKeyData
 	'app-state-sync-version': LTHashState
 	'lid-mapping': string
+	'user-devices': string[]
```

Comment:
```suggestion
	'device-list': string[]
```

can we call this device-list or something